includes/admin/views/Preview_template/default-preview-template.php

```diff
@@ -308,9 +308,10 @@
 				if ( $totals_arr ) :
 
 					foreach ( $totals_arr as $total ) :
+						$modified_label = apply_filters( 'custom_order_total_label', $total['label'], $key, $order );
```

Comment:
@gaminikhil Add the plugin prefix to the filter name and change it to something like wcdn_invoice_order_total_label.